src/main/java/org/zaproxy/zap/extension/jwt/ui/JWTOptionsPanel.java

```diff
@@ -232,18 +221,119 @@ private JPanel generalSettingsSection() {
         return generalSettingsPanel;
     }
 
+    private JPanel getFuzzerSettingsSection() {
+        JPanel fuzzerSettingsPanel = new JPanel(new GridBagLayout());
+        TitledBorder fuzzerSettingsBorder =
+                JWTUIUtils.getTitledBorder("jwt.settings.fuzzer.header");
+        fuzzerSettingsPanel.setBorder(fuzzerSettingsBorder);
+        GridBagConstraints gridBagConstraints = JWTUIUtils.getGridBagConstraints();
+        gridBagConstraints.gridy++;
+        fuzzerSettingsPanel.add(getHMACSignaturePanel(), gridBagConstraints);
+        gridBagConstraints.gridy++;
+        fuzzerSettingsPanel.add(getRSASignaturePanel(), gridBagConstraints);
+        return fuzzerSettingsPanel;
+    }
+
+    private JPanel getHMACSignaturePanel() {
+        JPanel hmacSignaturePanel = new JPanel();
+        hmacSignaturePanel.setLayout(new GridBagLayout());
+        hmacSignaturePanel.setSize(hmacSignaturePanel.getPreferredSize());
+        TitledBorder hmacSignaturePanelBorder =
+                JWTUIUtils.getTitledBorder("jwt.settings.fuzzer.hmac.signature.configuration");
+
+        hmacSignaturePanel.setBorder(hmacSignaturePanelBorder);
+        GridBagConstraints gridBagConstraints = JWTUIUtils.getGridBagConstraints();
+        JLabel jwtHmacPrivateKeyLabel = new JLabel("Key");
+        jwtHMacSignatureKey = new JPasswordField();
+        jwtHMacSignatureKey.setEditable(true);
+        jwtHMacSignatureKey.setColumns(15);
+        gridBagConstraints.gridx = 0;
+        hmacSignaturePanel.add(jwtHmacPrivateKeyLabel, gridBagConstraints);
+        gridBagConstraints.gridx++;
+        hmacSignaturePanel.add(jwtHMacSignatureKey, gridBagConstraints);
+        gridBagConstraints.gridx++;
+        return hmacSignaturePanel;
+    }
+
+    private JPanel getRSASignaturePanel() {
+        JPanel rsaSignaturePanel = new JPanel();
+        rsaSignaturePanel.setLayout(new GridBagLayout());
+        rsaSignaturePanel.setSize(rsaSignaturePanel.getPreferredSize());
+        TitledBorder rsaSignaturePanelBorder =
+                JWTUIUtils.getTitledBorder("jwt.settings.fuzzer.rsa.signature.configuration");
+        rsaSignaturePanel.setBorder(rsaSignaturePanelBorder);
+        GridBagConstraints gridBagConstraints = JWTUIUtils.getGridBagConstraints();
+        gridBagConstraints.gridx = 0;
+        JLabel jwtRsaPrivateKeyLabel = new JLabel("Private Key");
+        JButton jwtRsaPrivateKeyFileChooserButton = new JButton("Select...");
```

Comment:
i18n names?

Reply:
oh ... thanks for finding this. Will do the changes.